flake.nix

```diff
@@ -3,7 +3,7 @@
   description = "A nixos module to make buildbot a proper Nix-CI.";
 
   inputs = {
-    nixpkgs.url = "github:Nixos/nixpkgs/nixos-unstable-small";
+    nixpkgs.url = "github:qowoz/nixpkgs/buildbot410";
```

Comment:
With this we should be able to add `buildbot` to nix/treefmt/flake-module.nix
and also remove:

```
[[tool.mypy.overrides]]
module = "buildbot.*"
ignore_missing_imports = true
```

from pyproject.toml.

Reply:
But this requires py.typed in the buildbot module.